command-line/test/Command/Address/InspectSpec.hs

```diff
@@ -15,66 +15,83 @@ import Test.Utils
 
 spec :: Spec
 spec = describeCmd [ "address", "inspect" ] $ do
-    specInspectAddress ["Byron", "none"]
+    specInspectAddress ["Byron", "none"] []
         "37btjrVyb4KEgoGCHJ7XFaJRLBRiVuvcrQWPpp4HeaxdTxhKwQjXHNKL4\
         \3NhXaQNa862BmxSFXZFKqPqbxRc3kCUeTRMwjJevFeCKokBG7A7num5Wh"
 
-    specInspectAddress ["Icarus", "none"]
+    specInspectAddress ["Byron", "address_index", "account_index"]
```

Comment:
Afais we have no unit test for a specific derivation path value? Would that be an interesting thing to have?